### PR TITLE
test: test PR do not merge

### PR DESCRIPTION
**Pull request by thecode**
# Pull Request

> First PR? don't hesitate, push it.

## Description

> Describe what you did and why.ffff

**Related issue (if any):** fixes #*issue_number_goes_here*

## Checklist

- [x] I have followed this repository's contributing guidelines.
- [x] I have used the conventional commits specification for my commit messages.
- [x] I have signed my commits.
- [x] I will adhere to the project's code of conduct.

## Additional information

> Anything else?
>

### CODE REVIEW

File: src/aioswitcher/api/__init__.py [hunk 1]
```diff
@@ -721,8 +721,9 @@ def get_command(
             target_temp = self._max_temp
 
         # verify the target temp and set minimum if we provided with lower number
```
Comment (TomerFi) [inline src/aioswitcher/api/__init__.py:723]: ttttt

File: src/aioswitcher/api/__init__.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def _get_command_length(self):
 
 
 @final
-class SwitcherApi:
+class SwitcherApi1: # MAKE sure this fail CI
```
Comment (TomerFi) [inline src/aioswitcher/api/__init__.py]:
```suggestion
class SwitcherApi1: # MAKE sure this fail CII
```

File: src/aioswitcher/api/__init__.py [hunk 3]
```diff
@@ -721,8 +721,9 @@ def get_command(
             target_temp = self._max_temp
 
         # verify the target temp and set minimum if we provided with lower number
-        elif target_temp < self._min_temp:
-            target_temp = self._min_temp
+        # DO NOT MERGE ONLY A TEST
```
Comment (thecode) [inline src/aioswitcher/api/__init__.py]:
```suggestion
        # DO NOT MERGE ONLY A TEST - still don't
```

File: src/aioswitcher/api/__init__.py [hunk 4]
```diff
@@ -721,8 +721,9 @@ def get_command(
             target_temp = self._max_temp
 
         # verify the target temp and set minimum if we provided with lower number
-        elif target_temp < self._min_temp:
-            target_temp = self._min_temp
+        # DO NOT MERGE ONLY A TEST - still don't
+        elif target_temp <= self._min_temp:
```
Comment (thecode) [inline src/aioswitcher/api/__init__.py]:
```suggestion
        elif target_temp <= self._min_temp: # another test
```

File: src/aioswitcher/api/__init__.py [hunk 5]
```diff
@@ -93,7 +93,7 @@ def _get_command_length(self):
 
 
 @final
-class SwitcherApi:
+class SwitcherApi1: # MAKE sure this fail CII
```
Comment (thecode) [inline src/aioswitcher/api/__init__.py]:
```suggestion
class SwitcherApi1: # MAKE sure this fail CII1
```

File: src/aioswitcher/api/__init__.py [hunk 6]
```diff
@@ -721,8 +721,9 @@ def get_command(
             target_temp = self._max_temp
 
         # verify the target temp and set minimum if we provided with lower number
-        elif target_temp < self._min_temp:
-            target_temp = self._min_temp
+        # DO NOT MERGE ONLY A TEST - still don't
+        elif target_temp <= self._min_temp: # another test
+            target_temp != self._min_temp
```
Comment (thecode) [inline src/aioswitcher/api/__init__.py]:
```suggestion
            target_temp != self._min_te1mp
```

**Comment by TomerFi** [comment]
@thecode do we still need this open?

**Pull request closed without merging**